lldb/test/Shell/SymbolFile/NativePDB/local-variables.cpp

```diff
@@ -1,5 +1,7 @@
 // clang-format off
 
+// XFAIL: target=x86_64-{{.*}}-windows{{.*}}
```

Comment:
As far as I understand, NativePDB tests only run for Windows host+Windows target, contrary to the tests from the PDB folder which we managed to run on Linux target as part of our effort here https://discourse.llvm.org/t/rfc-lldb-support-remote-run-of-shell-tests/80072/10.
It's surprising that they didn't pass with your config

```
PASS: lldb-shell :: SymbolFile/NativePDB/stack_unwinding01.cpp (533 of 543)
Exit Code: 0

Command Output (stdout):
--
# RUN: at line 4
'C:\Python312\python.exe' C:\lldb\test\Shell\helper\build.py --compiler=any --arch=64 --tools-dir=C:/build-lldb-native/./bin --libs-dir=C:/build-lldb-native/./lib --compiler=clang-cl --nodefaultlib -o C:\build-lldb-native\tools\lldb\test\Shell\SymbolFile\NativePDB\Output\stack_unwinding01.cpp.tmp.exe -- C:\lldb\test\Shell\SymbolFile\NativePDB\stack_unwinding01.cpp
# executed command: 'C:\Python312\python.exe' 'C:\lldb\test\Shell\helper\build.py' --compiler=any --arch=64 --tools-dir=C:/build-lldb-native/./bin --libs-dir=C:/build-lldb-native/./lib --compiler=clang-cl --nodefaultlib -o 'C:\build-lldb-native\tools\lldb\test\Shell\SymbolFile\NativePDB\Output\stack_unwinding01.cpp.tmp.exe' -- 'C:\lldb\test\Shell\SymbolFile\NativePDB\stack_unwinding01.cpp'
# .---command stdout------------
# | Cleaning stack_unwinding01.ilk
# | Cleaning stack_unwinding01.cpp.tmp.exe-stack_unwinding01.obj
# | Cleaning stack_unwinding01.cpp.tmp.pdb
# | Cleaning stack_unwinding01.cpp.tmp.exe
# | 
# | 
# | 
# | compiling stack_unwinding01.cpp -> stack_unwinding01.cpp.tmp.exe-stack_unwinding01.obj
# |   STDOUT:
# | 
# | 
# | 
# | 
# | linking stack_unwinding01.cpp.tmp.exe-stack_unwinding01.obj -> stack_unwinding01.cpp.tmp.exe
# |   STDOUT:
# | 
# `-----------------------------
# RUN: at line 5
env LLDB_USE_NATIVE_PDB_READER=1 c:\build-lldb-native\bin\lldb.exe --no-lldbinit -S C:/build-lldb-native/tools/lldb\test\Shell\lit-lldb-init-quiet -f C:\build-lldb-native\tools\lldb\test\Shell\SymbolFile\NativePDB\Output\stack_unwinding01.cpp.tmp.exe -s      C:\lldb\test\Shell\SymbolFile\NativePDB/Inputs/stack_unwinding01.lldbinit 2>&1 | c:\build-lldb-native\bin\filecheck.exe C:\lldb\test\Shell\SymbolFile\NativePDB\stack_unwinding01.cpp
# executed command: env LLDB_USE_NATIVE_PDB_READER=1 'c:\build-lldb-native\bin\lldb.exe' --no-lldbinit -S 'C:/build-lldb-native/tools/lldb\test\Shell\lit-lldb-init-quiet' -f 'C:\build-lldb-native\tools\lldb\test\Shell\SymbolFile\NativePDB\Output\stack_unwinding01.cpp.tmp.exe' -s 'C:\lldb\test\Shell\SymbolFile\NativePDB/Inputs/stack_unwinding01.lldbinit'
# note: command had no output on stdout or stderr
# executed command: 'c:\build-lldb-native\bin\filecheck.exe' 'C:\lldb\test\Shell\SymbolFile\NativePDB\stack_unwinding01.cpp'
# note: command had no output on stdout or stderr


```